src/pages/settings/settings.html

```diff
@@ -31,5 +31,12 @@
         <ion-option value="60000">60 Seconds</ion-option>
       </ion-select>
     </ion-item>
+    <ion-item>
+      <ion-label>Sort Departures...</ion-label>
+      <ion-select [(ngModel)]="departureSort">
```

Comment:
Thanks for doing this, made me research stuff.

Reply:
(I just copied it from somewhere else)